components/lib/password/Password.css

```diff
@@ -23,6 +24,21 @@
     transition: width 1s ease-in-out;
 }
 
+
+
 .p-fluid .p-password {
     display: flex;
 }
+
+.p-password-strength-weak
+{
+    color: red !important;
+}
+.p-password-strength-medium
+{
+    color: orange !important;
+}
+.p-password-strength-strong
+{
+    color: green !important;
```

Comment:
Revert all the changes in this file the changes will go in the `theme` themselves so it can be colored per theme.